hw/ip/csrng/data/csrng.hjson

```diff
@@ -179,6 +179,8 @@
       desc: "Application interface command status register",
       swaccess: "ro",
       hwaccess: "hwo",
+      tags: [// Internal HW can modify status register
+                 "excl:CsrAllTests:CsrExclCheck"]
```

Comment:
would it be enough to exclude all `CsrNonInitTests` instead of `CsrAllTests`?

Reply:
No idea. These codes are not explained very well as far as I have searched. Assuming, that only a basic reset test will actually check the reset value, and all other tests will ignore, I guess this CsrNonIinitTests tag might work ok.

Reply:
Yes, this means that the exclusion will apply to all tests other than the `HwReset` test.

There is some documentation on this here:
https://docs.opentitan.org/doc/rm/register_tool/#verification-tags-definition-and-format